lib/src/main/kotlin/at/bitfire/vcard4android/contactrow/EventHandler.kt

```diff
@@ -27,15 +29,22 @@ object EventHandler: DataRowHandler() {
         super.handle(values, contact)
 
         val dateStr = values.getAsString(Event.START_DATE) ?: return
-        var full: LocalDate? = null
+        var full: Temporal? = null
         var partial: PartialDate? = null
         try {
             full = LocalDate.parse(dateStr)
-        } catch(e: DateTimeParseException) {
+        } catch(_: DateTimeParseException) {
             try {
-                partial = PartialDate.parse(dateStr)
-            } catch (e: IllegalArgumentException) {
-                logger.log(Level.WARNING, "Couldn't parse birthday/anniversary date from database", e)
+                // Some server providers (e.g. t-mobile) include time information
+                // This is allowed by RFC2426: https://www.rfc-editor.org/rfc/rfc2426#section-3.1.5
+                // And RFC6350: https://www.rfc-editor.org/rfc/rfc6350#section-6.2.5
```

Comment:
This code is not related to vCard, as it only handles the field of the local contacts provider. So RFCs about vCard don't apply.